tests/integration/test_end_to_end.py

```diff
@@ -0,0 +1,107 @@
+import pytest
```

Comment:
maybe some instructions (e.g. in a module docstring, or the readme) on the requirements for running this